modules/core/src/main/scala/scala/build/internals/EnvVar.scala

```diff
@@ -124,6 +153,40 @@ object EnvVar {
     val sparkHome = EnvVar("SPARK_HOME", "Spark installation directory", requiresPower = true)
   }
 
+  object Bloop extends EnvVarGroup {
+    override def groupName: String = "Bloop"
+    override def all = Seq(
+      bloopComputationCores,
+      bloopDaemonDir,
+      bloopExportJarClassifiers,
+      bloopJavaOpts,
+      bloopModule,
+      bloopPort,
+      bloopScalaVersion,
+      bloopVersion,
+      bloopServer,
+      bloopExtraTimeout,
+      bloopTravis
+    )
+    val bloopComputationCores = EnvVar(
+      "BLOOP_COMPUTATION_CORES",
+      "Number of computation cores to be used",
+      requiresPower = true
+    )
+    val bloopDaemonDir = EnvVar("BLOOP_DAEMON_DIR", "Bloop daemon directory", requiresPower = true)
+    val bloopExportJarClassifiers =
+      EnvVar("BLOOP_EXPORT_JAR_CLASSIFIERS", "Export jar classifiers", requiresPower = true)
+    val bloopJavaOpts = EnvVar("BLOOP_JAVA_OPTS", "Bloop Java options", requiresPower = true)
+    val bloopModule   = EnvVar("BLOOP_MODULE", "Bloop default module", requiresPower = true)
+    val bloopPort     = EnvVar("BLOOP_PORT", "Bloop default port", requiresPower = true)
+    val bloopScalaVersion =
+      EnvVar("BLOOP_SCALA_VERSION", "Bloop default Scala version", requiresPower = true)
+    val bloopVersion      = EnvVar("BLOOP_VERSION", "Bloop default version", requiresPower = true)
+    val bloopServer       = EnvVar("BLOOP_SERVER", "Bloop default host", requiresPower = true)
+    val bloopExtraTimeout = EnvVar("SCALA_CLI_EXTRA_TIMEOUT", "Extra timeout", requiresPower = true)
+    val bloopTravis       = EnvVar("TRAVIS", "Travis CI toggle", requiresPower = true)
```

Comment:
I don't think this should be promoted. Pretty sure it's only used for Bloop tests (and not for a while) and I would just remove it

Reply:
removed